AppController/terminate.rb

```diff
@@ -1,5 +1,5 @@
 $:.unshift File.join(File.dirname(__FILE__), "lib")
-APPSCALE_HOME = ENV['APPSCALE_HOME']
+CONFIG_FILE_LOCATION = "/etc/appscale"
```

Comment:
We have different names in different places. Djinn uses CONFIG_FILE_LOCATION ... I could try to uniform the name if you think this PR is good for it.